stl/inc/yvals_core.h

```diff
@@ -554,7 +555,8 @@
 #endif // ^^^ inline (not constexpr) in C++17 and earlier ^^^
 
 // Functions that became constexpr in C++20 via P0784R7
-#if _HAS_CXX20 && defined(__cpp_constexpr_dynamic_alloc)
+#if _HAS_CXX20 && defined(__cpp_constexpr_dynamic_alloc) \
+    && (defined(__clang__) || defined(__EDG__)) // TRANSITION, MSVC support for constexpr dynamic allocation

```

Comment:
All thisconstexpr stuf makes my head spin. As far as I know, we only have the constexpr destructor test that fails with MSVC.

What happens when we define this unconditionally?

Reply:
As far as I understand, this was applied because of the constexpr destructors only.
If defined unconditionally, most of the test cases in `P0784R7_library_support_for_more_constexpr_containers` will fail for MSVC because they rely on the constexpr destructors.

Reply:
Since we can't enable full `_CONSTEXPR20_DYNALLOC` for MSVC for `string`/`vector` due to the `constexpr` destructor issue, the idea was to just consider MSVC as not supporting `constexpr` dynamic allocation just yet and turn everything on at the same time.

Reply:
Thanks, I forgot about the destructor bug